packages/frontend-2/components/workspace/sidebar/Members.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <LayoutSidebarMenuGroup
+    :title="collapsible ? 'Members' : undefined"
+    :collapsible="collapsible"
+    icon="edit"
+    :icon-click="() => openSettingsDialog(SettingMenuKeys.Workspace.Members)"
+    icon-text="Edit team"
+    :tag="workspaceInfo.team.totalCount.toString() || undefined"
+    no-hover
+  >
+    <div
+      v-if="!isWorkspaceGuest"
+      class="flex lg:flex-col items-center lg:items-start justify-between lg:justify-start gap-2 pb-0 lg:pb-4 mt-1"
+    >
+      <div class="flex items-center gap-1">
+        <UserAvatarGroup
+          :overlap="false"
+          :users="team.map((teamMember) => teamMember.user)"
+          :max-avatars="3"
+        />
+        <button
```

Comment:
If you mean `CommonBadge`, we could but it doesn't have a dashed variant, it also doesn't have a click event on the whole thing, so i'd have to wrap it in a button anyway. It would also be purple so i'd have to use the color-classes to style it. Seems like a lot of overwrites for no real benefit that I can see.